ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -91,14 +91,9 @@ object LedgerApiTestTool {
 
     val defaultTests: Vector[LedgerTestSuite] = Tests.default(
       timeoutScaleFactor = config.timeoutScaleFactor,
-      ledgerClockGranularity = config.ledgerClockGranularity,
-      staticTime = config.staticTime,
-    )
-    val optionalTests: Vector[LedgerTestSuite] = Tests.optional(
-      timeoutScaleFactor = config.timeoutScaleFactor,
-      ledgerClockGranularity = config.ledgerClockGranularity,
       staticTime = config.staticTime,
     )
+    val optionalTests: Vector[LedgerTestSuite] = Tests.optional()
```

Comment:
Rather than allowing default parameters for these methods (which only makes sense because of a test, which is my fault), can we remove the parameters and inline the default values into `NamesSpec`?